frontend/js/servicecalls.js

```diff
@@ -298,6 +299,7 @@ window.serviceResetSettings = () => {
 
     vsync: true,
     autostart: false,
+    nv12: true,
```

Comment:
Because hyperhdr stable v20 release does not support it, yet, defaulting to `false` would be a more fitting default for now

Reply:
Would agree on that. In backend service it also defaults to false: https://github.com/webosbrew/hyperion-webos/blob/f45cff0cd39e9c908a22362f8bdf10a3246c08de/src/settings.c#L19